neurokit2/signal/signal_detrend.py

```diff
@@ -15,6 +15,7 @@ def signal_detrend(
     window=1.5,
     stepsize=0.02,
     components=[-1],
+    sampling_rate=None,
```

Comment:
we should put it to 1000 by default given that default stepsize is not an integer, 

also we should probably pipe down the sampling rate in the routines where we use it (ecg_process?)

Reply:
currently there don't seem to be any functions that call the locreg method of detrending (or have a parameter for specifying the method of detrending), other than intervals_process which I changed